src/main/Duke/virtue/VirtueFileReader.java

```diff
@@ -0,0 +1,38 @@
+package virtue;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class VirtueFileReader {
```

Comment:
It would be nice to explain what the file reader does for the application in a comment. Might be more important when in the future we deal with different files with more than one purpose